python/paddle/fluid/initializer.py

```diff
@@ -1033,6 +1034,49 @@ def _global_bias_initializer():
     return _global_bias_initializer_
 
 
+def calculate_gain(nonlinearity, param=None):
+    """
+    Get the recommended gain value of some nonlinearity function.
+
+    Args:
+        nonlinearity: nonlinearity function.
+        param: optional parameter for somme nonlinearity function
+
+    Returns:
+        The recommended gain value for nonlinearity function.
+
+    Examples:
+        .. code-block:: python
+
+            import paddle
+            gain = paddle.nn.initializer.calculate_gain('tanh')
+            initializer = paddle.nn.initializer.Orthogonal(gain)
+
+    """
+    if param is None:
+        param = 0.01
+    else:
+        assert isinstance(param, (bool, int, float))
+    recommended_gain = {
```

Comment:
shall we support calculate gain of selu?

Reply:
done